sourcecode/apis/contentauthor/app/Http/Controllers/H5PController.php

```diff
@@ -690,18 +691,21 @@ public function getVideo($videoId)
         return $videoAdapter->getVideo($videoId);
     }
 
-    public function getCopyright(H5PContent $h5p)
+    public function getCopyright(H5PContent $h5p, H5PCopyright $copyright, CacheRepository $cache)
     {
-        $copyrights = (resolve(H5PCopyright::class))->getCopyrights($h5p);
+        $copyrights = $cache->sear($h5p->getCopyrightCacheKey(), fn() => $copyright->getCopyrights($h5p));
```

Comment:
Prefer using `rememberForever` over `sear`, it's more descriptive when reading the code, and all `sear` does is call `rememberForever`.
Also used in `getInfo()`